src/store/account.ts

```diff
@@ -45,6 +45,7 @@ export const mutations = mutationTree(state, {
     if (state.address !== undefined) {
       if (name.length < 1) {
         name = getNameFromAddress(state.address);
+        window.localStorage.removeItem(state.address);
```

Comment:
Why do we save the address in the first place?

Reply:
It's an account name, which the user can change.